.github/workflows/e2e-component-pipeline.yml

```diff
@@ -0,0 +1,180 @@
+---
+name: Component-Pipeline
+on:
+  workflow_dispatch:
+    inputs:
+      e2eTestImage:
+        default: 'litmuschaos/litmus-e2e:ci'
+      goExperimentImage:
+        default: 'litmuschaos/go-runner:ci'
+      newLibImage:
+        default: 'litmuschaos/go-runner:ci'
+      oldLibImage:
+        default: 'litmuschaos/go-runner:latest'        
+      operatorImage:
+        default: 'litmuschaos/chaos-operator:ci'
+      runnerImage:
+        default: 'litmuschaos/chaos-runner:ci'
+      chaosNamespace:
+        default: 'litmus'
+      imagePullPolicy:
+        default: 'Always'
+      experimentImagePullPolicy:
+        default: 'Always'
+      updateWebsite: 
+        default: 'false'
+        
```

Comment:
We can also provide default working_directory as defaults for all jobs.
Ref:- https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#defaultsrun

Reply:
Yes, it's good to set on job level thanks!

Reply:
We can also give the default working_directory at the top level for all jobs.